src/components/foodDescription/FoodDescription.js

```diff
@@ -18,10 +21,31 @@ function FoodDescription(props) {
     const { foodList } = props;
     const keys = useMemo(() => _keys(foodList),[foodList])
 
+    const foodListSectionRefs = useRef([])
+
+    function setRef(ele){
```

Comment:
Take these functions outside

Reply:
i cannot take that function because that uses the ref in the scope